### DISCUSSION TITLE
Gerbera home page loads with errors, can't login or access server

### DISCUSSION THREAD

**Discussion by BobManMasher** [gerbera/gerbera#2219] [Q&A] [answered]
Having got 1.9.2 to compile and run on debian 10, I have struggled to get it to install on other systems because I can't get suitable dependency modules any more. A debian 11 OS is now available along with a version of gerbera 1.9.0 (handy because I am getting a peculiar dependency erro with my own compiled version. Gerbera 1.9.0 is installed and running happily but the web interface no matter what browser or platform gives an error when loading the home page, doesn't create it's cookies, sign in, detect the correct name of the server or otherwise function. There are errors in the page load as follows:

`1) popper.js:5727 Uncaught SyntaxError: Unexpected token 'export''

'2) jquery-3.5.1.min.js:2 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
send @ jquery-3.5.1.min.js:2'

'3) jquery-3.5.1.min.js:2 Failed to load resource: net::ERR_CONTENT_LENGTH_MISMATCH
send @ jquery-3.5.1.min.js:2'

'4) gerbera-updates.module.js:52 The system encountered an error`

I have checked permissions etc and compared all the settings and file structures with my functioning gerbera servers. I am clearly missing something but can't see what so any pointers would be great.

Thanks - Bob

**Comment by KarlStraussberger** [comment]
You may try `scripts/debian/build-deb.sh HEAD`. It builds all the libs. Just make sure the no other (dev) versions are installed (usr/include is searched before usr/local/include)

**Reply by BobManMasher** [reply]
Thanks Karl - still can't spot anything :(

BTW my 1.92 build won't install as it wants duktape203 which was on the buster based system it was compiled on but .. Bullseye comes with duktape205 which is >=2.0.3 and on my new Bullseye system. Is there a problem with the dependencies? - This is why I am trying to install V1.9.0 which is currently available through APT.

**Comment by BobManMasher** [comment, marked as answer]
I think this was an access rights issue with the way the gerbera user is created and where the js files are located.  My compiled version of 1.9.2 doesn't have the problem.

~As for the script - never knew it was there and it looks great though I didn't use it in the end as I had compiled new versions of some of the libs and options.

Thanks for your help - Bob
